# This service allows to load a trajectory from disk

# File to read from
# use package://package_name/file.json to load from a specific ROS package
string file_name

---

bool success
string error

# Trajectory
moveit_msgs/RobotTrajectory trajectory